src/main/scala/com/typesafe/sbt/license/TargetLanguage.scala

```diff
@@ -77,3 +77,19 @@ case object Html extends TargetLanguage {
 
   def htmlEncode(s: String) = org.apache.commons.lang3.StringEscapeUtils.escapeHtml4(s)
 }
+
+case object Csv extends TargetLanguage {
+  val ext = "csv"
+  def documentStart(title: String, reportStyleRules: Option[String]): String = ""
+  def documentEnd(): String = ""
+  def createHyperLink(link: String, content: String): String = s"$content"
```

Comment:
I guess my discomfort with the idea comes down to two things:

* CSV is a data format, and in a data format different data items should be separated.
* If someone were to browse the data in a spreadsheet, the hyperlinks would make it difficult to size one of the columns. And if someone wanted to copy/paste the text or the hyperlink, he/she would have to drill into the cell and carefully highlight just the part he/she wants.  (This goes back to my previous bullet point.)